.github/workflows/stale.yml

```diff
@@ -10,6 +10,7 @@ jobs:
     steps:
     - uses: actions/stale@v3
       with:
+        repo-token: ${{ secrets.WEAVEWORKSBOT_TOKEN }}
```

Comment:
```suggestion
        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

This is automatically included with all action runs and it should have write permission

Reply:
ah good to know

Reply:
the `WEAVEWORKSBOT_TOKEN` is necessary sometimes because actions performed by the actions `GITHUB_TOKEN` can't trigger other actions, like check runs for example.

Reply:
good to know! thanks